src/templates/k8s/connect_logging.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+POD_NAME=$(kubectl get pods --namespace warnet-logging -l "app.kubernetes.io/name=grafana,app.kubernetes.io/instance=loki-grafana" -o jsonpath="{.items[0].metadata.name}")
+
+GRAFANA_PASSWORD=$(kubectl get secret --namespace warnet-logging loki-grafana -o jsonpath="{.data.admin-password}" | base64 --decode || true)
```

Comment:
Do we have to do this? Is there no way this could just be admin:password or admin:admin (which was the default on docker)?

Reply:
If we have to, perhaps the justfile could echo it out for us? Or perhaps it doesn't work until after the thing is up or something, IDK, I'm just lazy... (can also leave for future PR)

Reply:
Just does print out the password when you run `just connectlogging`.

Can look into changing it into an easy to remember default.